src/pages/DatasetCatalog.js

```diff
@@ -56,25 +56,18 @@ class DatasetCatalog extends Component {
   }
 
   async getDatasets() {
-
-    if (this.state.isResearcher && '?reviewProfile' === this.props.location.search) {
-      const researcher = await Researcher.getPropertiesByResearcherId(Storage.getCurrentUser().dacUserId);
-      if (researcher.completed !== 'true') {
-        this.props.history.push('profile');
-      }
-    } else {
-      let catalog = await DataSet.findDataSets(this.USER_ID);
-      catalog.forEach((row, index) => {
-        row.checked = false;
-        row.ix = index;
-      });
-
-      this.setState({
-        dataSetList: { catalog: catalog },
-        currentPage: 1
-      });
-    }
-  }
+    // const currentUserId = Storage.getCurrentUser().dacUserId;
+    // let catalog = await DataSet.findDataSets(currentUserId);
+    let catalog = await DataSet.findDataSets(this.USER_ID);
```

Comment:
The use of `this.USER_ID` is an unusual pattern here, I would prefer that we access the user id from something that seems more reliable. In `componentDidMount`, which should execute before this method, we set `this.currentUser`. I think this would read more consistently with the rest of the app if we used `this.currentUser.dacUserId`

Reply:
Okay, that makes sense!